src/search.coffee

```diff
@@ -90,7 +90,8 @@ class Search
         else
           invalidId = true
       if invalidId
-        return callback new Error 'Unable to parse an ID out of result URL.'
+        return callback new Error 'Unable to parse an ID out of result URL.',
+          url: result.url or result.hostPageUrl
```

Comment:
Close! This is a subtle bug. The `Error` constructor just takes the one argument. Using `extend` is how you add instance-level properties in one line. Otherwise, just use multiple lines.

```coffee
coffee> e = new Error 'Here is the err msg', {'url'}
coffee> e.url
undefined
coffee> {extend} = require 'underscore'
coffee> e2 = extend new Error('some message'), {'url'}
coffee> e2.url
'url'
```